datasets/conllpp/conllpp.py

```diff
@@ -0,0 +1,229 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""CrossWeigh: Training Named Entity Tagger from Imperfect Annotations"""
+
+import logging
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{wang2019crossweigh,
+  title={CrossWeigh: Training Named Entity Tagger from Imperfect Annotations},
+  author={Wang, Zihan and Shang, Jingbo and Liu, Liyuan and Lu, Lihao and Liu, Jiacheng and Han, Jiawei},
+  booktitle={Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP)},
+  pages={5157--5166},
+  year={2019}
+}
+"""
+
+_DESCRIPTION = """\
+CoNLLpp is a corrected version of the CoNLL2003 NER dataset where labels of 5.38% of the sentences in the test set
+have been manually corrected. The training set and development set are included for completeness.
+For more details see https://www.aclweb.org/anthology/D19-1519/ and https://github.com/ZihanWangKi/CrossWeigh
+"""
+
+_URL = "https://github.com/ZihanWangKi/CrossWeigh/raw/master/data/"
+_TRAINING_FILE = "conllpp_train.txt"
+_DEV_FILE = "conllpp_dev.txt"
+_TEST_FILE = "conllpp_test.txt"
+
+
+class ConllppConfig(datasets.BuilderConfig):
+    """BuilderConfig for Conll2003"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig forConll2003.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(ConllppConfig, self).__init__(**kwargs)
+
+
+class Conll2003(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Conllpp(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `conllpp`.
You'll need to regenerate the dataset_infos.json file after this change